CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+Fortigate exporter uses GitHub to manage reviews of pull requests.
+
+* If you are a new contributor see: [Steps to Contribute](#steps-to-contribute)
+
+* If you have a trivial fix or improvement, go ahead and create a pull request,
+  addressing it to @bluecmd
+
+* If you plan to do something more impacting, first discuss your ideas
+  on our [IRC](https://matrix.to/#/#fortigate_exporter:matrix.org)
+  This will avoid unnecessary work and surely give you and us a good deal
+  of inspiration.
+
+* Relevant coding style guidelines are the [Go Code Review
+  Comments](https://code.google.com/p/go-wiki/wiki/CodeReviewComments)
+  and the _Formatting and style_ section of Peter Bourgon's [Go: Best
+  Practices for Production
+  Environments](https://peter.bourgon.org/go-in-production/#formatting-and-style).
```

Comment:
Did we use that before? I read the points that section adds and I don't think it's worth pulling in them as a source for the three or so extra things they added that I don't even think affects us right now.

Reply:
```suggestion
```
Sure . I was not confident enough to remove it myself and this is always easier to remove than to add ...